zerver/openapi/zulip.yaml

```diff
@@ -18578,6 +18614,14 @@ components:
           type: string
           description: |
             The name of the stream.
+        archived:
+          type: boolean
+          description: |
+            A boolean indicating whether the stream is deactivated;
+            true if the stream is deactivated, false otherwise.
+
+            **Changes**: Changed in Zulip 9.0 (feature level 242).
+            Previously, only active streams were sent.
```

Comment:
I suggest the following:

> New in Zulip 9.0 (feature level 242). Previously, this API never returned archived streams.

p.s. you could have guessed this if you looked at out conventions for other `Changes` entries.

Reply:
Done.